vyper_fetch.sh

```diff
@@ -1,10 +1,16 @@
 #!/bin/bash
```

Comment:
can we move all files related to building vyper into separate folder? like `build_vyper` or something
the list can still be in the root